net/net.sh

```diff
@@ -26,17 +26,20 @@ Operate a configured testnet
  logs     - Fetch remote logs from each network node
 
  start/update-specific options:
-   -T [tarFilename]            - Deploy the specified release tarball
-   -t edge|beta|stable|vX.Y.Z  - Deploy the latest tarball release for the
-                                 specified release channel (edge|beta|stable) or release tag
-                                 (vX.Y.Z)
-   -i update_manifest_keypair  - Deploy the tarball using 'solana-install deploy ...'
-                                 (-t option must be supplied as well)
-   -f [cargoFeatures]          - List of |cargo --feaures=| to activate
-                                 (ignored if -s or -S is specified)
-   -r                          - Reuse existing node/ledger configuration from a
-                                 previous |start| (ie, don't run ./multinode-demo/setup.sh).
-   -D /path/to/programs        - Deploy custom programs from this location
+   -T [tarFilename]                    - Deploy the specified release tarball
+   -t edge|beta|stable|vX.Y.Z          - Deploy the latest tarball release for the
+                                        specified release channel (edge|beta|stable) or release tag
+                                        (vX.Y.Z)
+   -i update_manifest_keypair          - Deploy the tarball using 'solana-install deploy ...'
+                                         (-t option must be supplied as well)
+   -f [cargoFeatures]                  - List of |cargo --feaures=| to activate
+                                        (ignored if -s or -S is specified)
+   -r                                  - Reuse existing node/ledger configuration from a
+                                         previous |start| (ie, don't run ./multinode-demo/setup.sh).
+   -D /path/to/instruction-processors  - Deploy custom programs from this location
+   -c [number]                         - Number of solana-bench-tps clients to start
+   -e [number]                         - Number of solana-bench-exchange clients to start
```

Comment:
It would be nice if we left room for easy future expansion here, I imagine we'll have other client types over time.  How about:
1) Drop `-e`, and retain just a `-c` (but with a different format, see below)
2) If `-c` is not specified, default to all bench-tps clients (so existing scripts that use `net.sh` don't need an immediate update)
3) `-c` is specified multiple times, once for each client type: `-c tps=3 -c exchange=5`.

Reply:
@mvines wdyt now?